src/utils/api.js

```diff
@@ -0,0 +1,52 @@
+import axios from 'axios';
+import config from './config';
+// import { getToken } from './auth';
+
+
+// const token = getToken();
+const token = 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6MSwidXNlcm5hbWUiOiJpbWlyZSIsImVtYWlsIjoiaW1pcmVAZ21haWwuY29tIiwiaWF0IjoxNTQxNTgyMjQzLCJleHAiOjE1NDE1ODk0NDN9.KqR2wO0Q94VjC3zr7CqwYYz-_6HrQY0j63jiBUILlcw';
```

Comment:
Hey @imireallan ,
I'm sure this `token` is just a mockup of the server behaviour right?

Reply:
yes @SrMoffat am using that way because the login functionality is yet to be finished

Reply:
@PaulKariukiRimiru the token is just for testing because the sign up and login is yet to be merged